test/explorer/polars_backend/decimal_unstable_test.exs

```diff
@@ -23,29 +23,14 @@ defmodule Explorer.PolarsBackend.DecimalUnstableTest do
   end
 
   test "mean returns decimal instead of float", %{df: df} do
-    assert_raise(
-      RuntimeError,
-      """
-      DataFrame mismatch.
-
-      expected:
-
-          names: ["a"]
-          dtypes: %{"a" => {:f, 64}}
-
-      got:
-
-          names: ["a"]
-          dtypes: %{"a" => {:decimal, 38, 1}}
-      """,
-      fn -> DF.summarise(df, a: mean(a)) end
-    )
+    # shoudl return decimal, instead returns float
+    assert [1.5] == df |> DF.summarise(a: mean(a)) |> DF.pull(:a) |> Explorer.Series.to_list()
```

Comment:
I actually think we can get rid of this whole file. From the moduledoc:

> If the tests in the module start breaking, it probably means Polars has changed its decimal implementation. Be prepared to change the tests: they function as canaries rather than imposing expected behavior.

Both tests are "failing" in such a way that makes me think the specific decimal instabilities we were fixed.